servers/zms/schema/zms_server.sql

```diff
@@ -37,7 +37,8 @@ CREATE TABLE IF NOT EXISTS `zms_server`.`domain` (
   INDEX `idx_modified` (`modified` ASC),
   INDEX `idx_account` (`account` ASC),
   INDEX `idx_ypmid` (`ypm_id` ASC))
-ENGINE = InnoDB;
+ENGINE=InnoDB DEFAULT CHARSET=utf8 ROW_FORMAT=DYNAMIC;
+-- Prevent ERROR 1709 (HY000)
```

Comment:
Can you please provide additional details as far as what is causing this error. 

btw, this is auto generated file so it cannot be modified. if necessary changes should be done in the workbench file and generate a new version of the sql script.